anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/individuals/IndividualsControlChartAnomalyDetector.java

```diff
@@ -118,8 +118,8 @@ public AnomalyResult classify(MetricData metricData) {
         val thresholds = new AnomalyThresholds(
                 this.mean + strongDelta,
                 this.mean + strongDelta,
-                this.mean - strongDelta,
-                this.mean - strongDelta);
+                this.mean - strongDelta, this.mean - strongDelta
```

Comment:
See above about indent. Also should these all be strong (I assume so but just checking).

Reply:
Not sure about the strong thing--let me take a look. (It was there already but it could be a bug.)

Reply:
Fixed the formatting thing. Also yeah looks like all strong is intentional.